Makefile

```diff
@@ -7,4 +7,7 @@ install-tools:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(TOOLS_BIN_DIR) v1.50.1
 
 lint: install-tools checklicense impi
-	${LINTER} run ./...
\ No newline at end of file
+	${LINTER} run ./...
+
+build-check:
+	go test ./test/metric_value_benchmark -p 1 -timeout 30m -computeType=EC2 -v --tags=integration
```

Comment:
Is there any reason we can't run `go build`? This tries to execute the actual integration tests and it's going to fail.

Reply:
go build ignores `*_test.go` files, so I went with `go test -c` instead

Reply:
We can use go_build after this PR https://github.com/aws/amazon-cloudwatch-agent-test/pull/109. For now, temporarily support is fine with me.

Reply:
I haven't read through #109 yet. What action did you make to support test files with `go build`? And what benefit does switching from `go test -c` to `go build` give us?